scripts/helm/ort-server/values.yaml

```diff
@@ -20,13 +20,14 @@ vault:
 #######################################################
 
 keycloak:
-  command: ["/opt/keycloak_init/init-keycloak.sh", "/opt/bitnami/keycloak/bin/kc.sh", "start-dev"]
+  image:
+    tag: 26.1.4-debian-12-r1
```

Comment:
Is there a reason to use this specific version and not just 26.2.0 like in Compose?